src/cli/onefuzz/templates/regression.py

```diff
@@ -0,0 +1,250 @@
+        check_retry_count: Optional[int] = None,
+        check_fuzzer_help: bool = True,
+        fail_on_repro: bool = False,
+    ) -> Optional[Job]:
+
+        return self._create_job(
+            TaskType.libfuzzer_regression,
+            project,
+            name,
+            build,
+            pool_name,
+            crashes,
+            input_reports,
+            inputs,
+            target_exe,
+            tags,
+            notification_config,
+            target_env,
+            setup_dir,
+            reboot_after_setup,
+            target_options,
+            dryrun,
+            duration,
+            report_list,
+            crash_report_timeout,
+            debug,
+            check_retry_count,
+            check_fuzzer_help,
+            fail_on_repro,
+        )
+
+    def _create_job(
+        self,
+        task_type: TaskType,
+        project: str,
+        name: str,
+        build: str,
+        pool_name: str,
+        crashes: Container = None,
+        input_reports: Container = None,
+        inputs: Optional[Directory] = None,
+        target_exe: File = File("fuzz.exe"),
+        tags: Optional[Dict[str, str]] = None,
+        notification_config: Optional[NotificationConfig] = None,
+        target_env: Optional[Dict[str, str]] = None,
+        setup_dir: Optional[Directory] = None,
+        reboot_after_setup: bool = False,
+        target_options: Optional[List[str]] = None,
+        dryrun: bool = False,
+        duration: int = 24,
+        report_list: Optional[List[str]] = None,
+        crash_report_timeout: Optional[int] = None,
+        debug: Optional[List[TaskDebugFlag]] = None,
+        check_retry_count: Optional[int] = None,
+        check_fuzzer_help: bool = True,
+        fail_on_repro: bool = False,
+    ) -> Optional[Job]:
+
+        if not ((crashes and input_reports) or inputs):
+            self.logger.error(
+                "please specify either the 'crash' and 'input_reports' parameters or the inputs parameter"
+            )
+
+        if dryrun:
+            return None
+
+        self.logger.info("creating regression task from template")
+
+        helper = JobHelper(
+            self.onefuzz,
+            self.logger,
+            project,
+            name,
+            build,
+            duration,
+            pool_name=pool_name,
+            target_exe=target_exe,
+        )
+
+        helper.add_tags(tags)
+        if crashes:
+            if self.onefuzz.containers.get(crashes):
+                helper.define_containers(ContainerType.unique_inputs)
+            else:
+                self.logger.error(f"invalid crash container {crashes}")
+
+        if input_reports:
+            if self.onefuzz.containers.get(input_reports):
+                helper.define_containers(ContainerType.input_reports)
+            else:
+                self.logger.error(f"invalid crash container {input_reports}")
+
+        if inputs:
```

Comment:
I wonder if we should make the container used for this more specific to the effort at hand, and maybe even unique to this instance of the task.

Here's my thought.  I have a target `foo` that I've been fuzzing for for a while.  A 3rd party sends me a potentially crashing input for `foo`.  I want to see if this new input is crashing.

As is, this will upload said new input into the container "unique_inputs+UUID(project + name)", which pollutes the unqiue_inputs container with this potentially crashing input.

Reply:
updated the code to generate a unique name for the input container